### ISSUE TITLE
TypeError: Cannot read properties of undefined (reading 'getTime')

### ISSUE THREAD

**Issue by chaoming** [firebase/firebase-tools#8008]
Firebase-tools and Node v23 issue:
```
TypeError: Cannot read properties of undefined (reading 'getTime')
```

### [REQUIRED] Environment info

**firebase-tools:** 11.30.0

**Platform:** maOS

### [REQUIRED] Steps to reproduce

Run the Firebase emulators including hosting. Visit the local hosting port.

How can I upgrade the superstatic version in the firebase-tools? superstatic v9.1.0 fixes the issue.

**Comment by aalej** [comment]
Hey @chaoming, thanks for reaching out. This issue should be fixed in firebase-tools [v13.26.0](https://github.com/firebase/firebase-tools/releases/tag/v13.26.0) where the `superstatic` dependency was updated to 9.1.0. Please upgrade to firebase-tools >= v13.26.0 to resolve the issue.  Also, I'd recommend using an even version of Node(20.x.x or 22.x.x) since those versions have long term support

**Comment by chaoming** [comment]
@aalej thank you. that works.
